polaris-react/src/components/Scrollable/Scrollable.scss

```diff
@@ -21,6 +23,62 @@
       var(--p-color-border-interactive-focus);
     outline-offset: var(--p-space-05);
   }
+
+  &.hasTopShadow,
+  &.hasBottomShadow {
+    /* Why not combine the box shadows onto one element? Because then it would
+     * need to be height: 100%, but that doesn't work when there's no explicit
+     * height set on the parent element (eg; for the Modal component which uses
+     * min-height: 80vh instead). */
+    &::before,
+    &::after {
+      content: '';
+      pointer-events: none;
+
+      /* Using CSS sticky to position the pseudo elements relative to the scroll
+       * container insead of relative to the contents */
+      position: sticky;
+
+      /* left/float/margin-right ensures it does not take up any static space,
+       * but still abides by position: sticky rules (ie; sorta position: fixed,
+       * but relative to the scroll container). */
+      left: 0;
+      float: left;
+      // stylelint-disable-next-line -- Can't use a token value here
+      margin-right: -100%;
+
+      /* Full width, but only as high as the shadow needs to be */
+      width: 100%;
+      // stylelint-disable-next-line -- Using custom variable here for the height
+      height: var(--pc-scrollable-shadow-size);
+
+      /* Arbitrarily picked this because it sorta sits above other elements most
+      * of the time without obscuring modals that might be above, etc.
+      * See https://github.com/Shopify/polaris/pull/9668/files#diff-b4fd34abf294f8d6bdcf784136be2ac82883085b8b0f91e450b2210eb6caaef9R70 */
```

Comment:
```suggestion

      /* Arbitrarily picked this because it sorta sits above other elements most of the time without obscuring modals that might be above, etc.
      *
      * See https://github.com/Shopify/polaris/pull/9668/files#diff-b4fd34abf294f8d6bdcf784136be2ac82883085b8b0f91e450b2210eb6caaef9R70 */
```